### ISSUE TITLE
logo removal

### ISSUE THREAD

**Issue by fredj** [openlayers/openlayers#7508]
In #7329, Tim simplify the attributions handling. I want to do the same for the logos.
My proposal is to completely remove the logo support and use the attributions system instead.

Before:
```js
var source = new ol.source.XYZ({
  attributions: 'some attribution',
  logo: {
    src: "https://www.example.com/",
    href: "https://www.example.com/logo.png"
  }
});
```

After:
```js
var source = new ol.source.XYZ({
  attributions: [
    '<a href="https://www.example.com/"><img src="https://www.example.com/logo.png"></a>',
    'some attribution'
  ]
});
```

**Comment by fredj** [comment]
For the OpenLayers logo, I was thinking of adding a new `attributions` option to ol.control.Attribution to store "static" attributions (not related to a source, always displayed). Default to:
```
['<a href="https://www.openlayers.org/"><img src="base64....."></a>']
```
or a simple link without image (like leaflet).

And to remove the OpenLayers attribution, an empty array could be passed.

**Comment by ahocevar** [comment]
An `attributions` option to `ol.control.Attribution` sounds good. But I think the base64 image for OpenLayers should not be in the library js code, but in the css. Which would mean a default like
```js
['<a href="https://www.openlayers.org/"><div class="ol-logo"></div></a>']
```

**Comment by tschaub** [comment]
This sounds great @fredj.  We use `logo: false` to avoid the logo right now.  If a new option is added to the attribution control, it would be nice if `logo: false` as a map option could be used to determine the appropriate configuration for the attribution control.

**Comment by cs09g** [comment]
what if there're multiple layers visible and they have different attributions or logos?

**Comment by fredj** [comment]
> what if there're multiple layers visible and they have different attributions or logos?

same as now: the attributions or logos will be displayed as a list by `ol.control.Attribution`

**Comment by fredj** [comment]
fixed with #7615
